app/controllers/application_controller.rb

```diff
@@ -1,10 +1,9 @@
 class ApplicationController < ActionController::Base
   include ActionController::Serialization
 
-  def authenticate!
-    my_session = false
+  def authenticate!(providers = [])
```

Comment:
Hi @pachonk,

Thanks for this. Could you clarify for me where the providers as an array comes in when this method is called? I can't seem to find it anywhere.